src/QsCompiler/Tests.Compiler/TestCases/QirTests/ExecutionTests.qs

```diff
@@ -1048,11 +1614,59 @@
         Message("********************\n");
     }
 
+    @EntryPoint()
+    operation TestInterpolatedStrings() : String {
+
+        let arr1 = [1,2,3];
+        let arr2 = ["1","2","3"];
+        let arr3 = ["","2","","4"];
+        let tuple1 = (1,(2,3));
+        let tuple2 = ("1",("2","3"));
+        let tuple3 = ("1",("","3"));
+        let res = One;
+        let pauli = PauliX;
+        let pauliArr = [PauliY, PauliI];
+        let range = 0..-1..0;
+        let (fct, op) = (PrintSection, RunExample);
+        let udt = Tuple("Hello", "World");
+        
+        Message("simple string");
+        Message($"{"interpolated string"}");
+        // Todo: bigint is not yet supported in the runtime
+        Message($"{true} or {false}, {res == Zero ? false | true}, {res == One ? false | true}, {res == One ? true | false}, {res == Zero ? true | false}");
+        Message($"{1}, {-1}, {1 - 1}");
+        Message($"{1.}, {2.0}, {1e5}, {.1}, {-1.}, {1. - 1.}");
+        Message($"{Zero}, {res}");
+        Message($"{PauliZ}, {pauli}, {pauliArr[0]}, {pauliArr[1...]}");
+        Message($"{1..3}, {3..-1..1}, {range}");
+        Message($"{[1,2,3]}, {["1","2","3"]}, {arr1}, {arr2}, {arr3}");
+        Message($"{()}, {(1,(2,3))}, {("1",("2","3"))}, {tuple1}, {tuple2}, {tuple3}");
+        
+        use (q, qs) = (Qubit(), Qubit[3]);
+        Message($"{q}, {qs}");
+        Message($"{PrintSection}, {RunExample}, {(fct, op)}");
+        Message($"{udt::Item1}, {Foo(1)}, {udt}");
+        return "All good!";
+    }
+
+    @EntryPoint()
+    operation NoReturn() : Unit {}
+
+    @EntryPoint()
+    operation ReturnsUnit() : Unit {
+        return ();
+    }
+
+    @EntryPoint()
+    operation ReturnsString() : String {
+        return "Success!";
+    }
+
     @EntryPoint()
     operation RunExample() : Unit {
 
         PrintSection(1, "");
-
+        
```

Comment:
Nit: it seems like the editor might have added a few space characters.

Reply:
I'll wait to have a formatter to remove these automatically. :) They occur when editing multiple lines, e.g. commenting them out.